mass_mailing_list_dynamic/wizards/partner_merge.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+
+class BasePartnerMergeAutomaticWizard(models.TransientModel):
+    _inherit = "base.partner.merge.automatic.wizard"
+
+    def _merge(self, partner_ids, dst_partner=None, extra_checks=True):
+        return super(
+            BasePartnerMergeAutomaticWizard, self.with_context(
+                partner_merge_wizard=True
```

Comment:
Isn't there already any context to use?

Reply:
Not exists any value in context to re-use it here, changed to `bypass_dynamic_list_check`.